WalletWasabi.Tests/UnitTests/BlockchainAnalysis/CoinJoinAnonscoreTests.cs

```diff
@@ -195,7 +195,8 @@ public void InputMergePunishment()
 		[Fact]
 		public void InputMergePunishmentNoInheritance()
 		{
-			// Input merging results in worse inherited anonset, but not in worse anonset if there's nothing to inherit.
+			// Input merging results in worse inherited anonset, but does not punish gains from output indistinguishability
+``
```

Comment:
sorry, i accidentally deleted the last backtick in my suggestion and introduced this syntax error

Reply:
[`9c86406` (#4724)](https://github.com/zkSNACKs/WalletWasabi/pull/4724/commits/9c864064ece0c20be87e359da9d1f36fe31b4c60)